src/avs/AVSDetailsTVLTab.jsx

```diff
@@ -0,0 +1,273 @@
+            <Spinner color="primary" size="lg" />
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+const tokens = {
+  eth: {
+    name: 'Beacon',
+    symbol: 'ETH',
+    logo: '/eth.png'
+  },
+  lst: {
+    name: 'Liquid Staking Tokens',
+    logo: '/eth.png'
+  },
+  eigen: {
+    name: 'Eigen',
+    symbol: 'EIGEN',
+    logo: '/eigen.webp'
+  }
+};
+
+function TokensBreakdownList({ totalTokens, isAVSLoading, ethRate }) {
+  const sortedTotalTokens = useMemo(() => {
+    const arr = Object.entries(totalTokens);
+    arr.sort((a, b) => b[1] - a[1]);
+    return arr;
+  }, [totalTokens]);
+
+  const sum = useMemo(
+    () => sortedTotalTokens.reduce((acc, [_, total]) => acc + total, 0),
+    [sortedTotalTokens]
+  );
+
+  const compact = !useTailwindBreakpoint('md');
+
+  return (
+    <Table
+      aria-label="Breakdown of ETH, EIGEN and Liquid Staking Tokens"
+      layout="fixed"
+      hideHeader
+      classNames={{
+        wrapper: 'border border-outline rounded-lg',
+        tr: 'border-b border-outline last:border-none'
+      }}
+      topContent={
+        <div className="text-foreground-1 text-medium">Tokens distribution</div>
+      }
+    >
+      <TableHeader>
+        <TableColumn className="w-1/2">Token</TableColumn>
+        <TableColumn>Total</TableColumn>
+      </TableHeader>
+      <TableBody>
+        {isAVSLoading &&
+          [...new Array(3)].map((_, i) => (
+            <TableRow key={i}>
+              <TableCell className="pl-0">
+                <Skeleton className="h-5 w-full rounded-md" />
+              </TableCell>
+              <TableCell className="flex justify-end">
+                <Skeleton className="h-10 w-20 rounded-md" />
+              </TableCell>
+            </TableRow>
+          ))}
+        {!isAVSLoading &&
+          sortedTotalTokens.map(([key, total]) => (
+            <TableRow key={key}>
+              <TableCell className="text-sm pl-0">
+                <div className="flex truncate gap-x-2 items-center">
+                  <Image height={16} width={16} src={tokens[key].logo} />
+                  <span className="text-foreground-2 truncate">
+                    {tokens[key].name}
+                  </span>{' '}
+                  <span className="text-foreground-1">
+                    {tokens[key].symbol}
+                  </span>
+                  <span className="text-foreground-1">
+                    {((total / sum) * 100).toFixed(2)}%
+                  </span>
+                </div>
+              </TableCell>
+              <TableCell className="text-sm flex justify-end">
+                <div className="text-end">
+                  {key === 'eigen' ? (
+                    <EigenDisclaimer />
+                  ) : (
+                    <div>{formatUSD(total * ethRate, compact)}</div>
+                  )}
+
+                  <div className="text-xs text-foreground-2">
+                    {formatNumber(total, compact)} tokens
```

Comment:
"tokens" is redundant. If it's easy, we can show the respective token symbol. If not, you can just skip it.

```suggestion
                    {formatNumber(total, compact)}
```

Reply:
updated
<img width="587" alt="image" src="https://github.com/user-attachments/assets/1d30983c-6777-4422-8d6c-75ef18ba3c54">